### PR TITLE
Configuration for bantime increment added

### PR DESCRIPTION
**Pull request by tobiicerb**
I added the necessary parts for bantime increment that is available since version 0.11.1. Those options will only be added if the version is this or higher.

### CODE REVIEW

File: templates/etc/fail2ban/jail.local.j2 [hunk 1]
```diff
@@ -2,10 +2,27 @@
...
+bantime.maxtime = {{ fail2ban_bantime_maxtime }}
+{% endif %}
+bantime.factor = {{ fail2ban_bantime_factor }}
+{% if fail2ban_bantime_multipliers is not defined %}
+bantime.formula = {{ fail2ban_bantime_formula }}
+{% endif %}
+{% if fail2ban_bantime_multipliers is defined %}
+bantime.multipliers = {{ fail2ban_bantime_multipliers }}
+{% endif %}
+bantime.overalljails = {{ fail2ban_bantime_overalljails | lower }}
```
Comment (tersmitten) [inline templates/etc/fail2ban/jail.local.j2:24]: `false` will result in no value here, right?

Comment (tobiicerb) [reply templates/etc/fail2ban/jail.local.j2:24]: Yes. The idea is to not configure any bantime options if you don't have the minimum required version.

Comment (tersmitten) [reply templates/etc/fail2ban/jail.local.j2:24]: And probably should be casted (to `string` to use `lower`)?

Comment (tersmitten) [reply templates/etc/fail2ban/jail.local.j2:24]: In some other roles I do something like this:

```
{{ fail2ban_bantime_overalljails | bool | to_json }}
```

Comment (tobiicerb) [reply templates/etc/fail2ban/jail.local.j2:24]: Wait, was the question about `{{ fail2ban_bantime_overalljails }}` or about `{{ fail2ban_version }}`?

So for `fail2ban_version` my comment holds and if the check` {% if fail2ban_version is version('0.11.1', '>=') %} `  returns `false` than no bantime configuration will be added as earlier versions don't have those variables and I do not want to break anything.

For `fail2ban_bantime_overalljails` what will be added to the configuration file is the string `false`. I am completely fine with changing it to `{{ fail2ban_bantime_overalljails | bool | to_json }}` if this is the preferred way. I would do it for `fail2ban_bantime_increment` as well.

**Pull request merged**